### DISCUSSION TITLE
Contributing to and maintaining the install and user docs

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#210] [Pull requests]
I was recently looking at the [installation](https://linuxmint-installation-guide.readthedocs.io/en/latest/) and [user](https://linuxmint-user-guide.readthedocs.io/en/latest/) docs to share with a friend, and noticed they are a bit outdated and incomplete. They also have some old unaddressed open issues, and don't seem to be actively maintained.

I am interested in rewriting them, and then keeping them up-to date with each release. At my previous job at a high-school I installed and maintained Mint on around 100 PCs and laptops, and had to write a lot of internal guides. This gained me the sad ability to spot things which would bring questions my way, and thus spent an year rewording things until my office was silent.

Namely (and at a glance) I think the following could be some issues on the Installation Guide: 
- Lack of accessibility mentions 
- Lack of steps (in the ISO verification in particular)
- Outdated screenshots
- Lack of details on what disabling Secure Boot implies (W11)
- Lack of details on what to do if something goes wrong or choices are provided (user doesn't know what graphics driver to choose -> user skips searching and goes directly to ask it somewhere, or decides "linux doesn't work")
- Sounds silly, but the "Create the bootable media" section does not specify that the process will erase drive contents, and someone somewhere is bound to think that it creates a LM.iso file or something.

The User Guide right now only has information on a couple of different topics, so it would need to be greatly expanded (info on all the pre-installed applications, differences between different DE configurations, etc).

If I contribute, I would try to keep some of the current urls and structures the same, as to facilitate translation and not brake too many tutorials that rely on them. I am asking before doing anything because it's a lot of work, and I would not like to spend my free time on something that might not be welcomed. I assume that I would start by forking the projects, redoing them, getting them reviewed and hopefully merging them before the LM 22 release next year.

If anything in this comes as judgmental, inappropriate or rushed, please note that that is not my intent, as I have huge respects for this community (and English is not my first language). Thanks for all the work the LM team does, it was the second distro I tried as a kid, and the one that made me stay and eventually start a career around Linux.

_PS: because this docs aren't too actively maintained, I did not know who to contact for this. If someone reading this knows (but isn't them) I would greatly appreciate it if you could forward me. Thanks in advance._

**Comment by eco-star** [comment]
Hey mate, this is a good specification of my created idea a few days ago: https://github.com/orgs/linuxmint/discussions/204 :) I would help in the online wiki aswell.

**Reply by ghost** [reply]
Hello, sorry I missed that discussion. Yeah, that would be great, I guess we need to wait until a maintainer notices this. I speak Spanish and English, so that would already mean 5 languages that an improved wiki would be written in. That was actually my main concert with modifying anything in the install docs, since it is currently written in 24 languages.
